doc/userguide/performance/high-performance-config.rst

```diff
@@ -1,19 +1,341 @@
+    L3 cache:            46080K
+    NUMA node0 CPU(s):   0-17,36-53
+    NUMA node1 CPU(s):   18-35,54-71
+
+It is recommended that 36 worker threads are used and the NIC set up could be as follows:
+
+::
+
+    rmmod i40e && modprobe i40e
+    ifconfig eth1 down
+    /usr/local/sbin/ethtool -L eth1 combined 36
+    /usr/local/sbin/ethtool -K eth1 rxhash on
+    /usr/local/sbin/ethtool -K eth1 ntuple on
+    ifconfig eth1 up
+    ./set_irq_affinity local eth1
+    /usr/local/sbin/ethtool -X eth1 hkey 6D:5A:6D:5A:6D:5A:6D:5A:6D:5A:6D:5A:6D:5A:6D:5A:6D:5A:6D:5A:6D:5A:6D:5A:6D:5A:6D:5A:6D:5A:6D:5A:6D:5A:6D:5A:6D:5A:6D:5A:6D:5A:6D:5A:6D:5A:6D:5A:6D:5A:6D:5A equal 36
+    /usr/local/sbin/ethtool -A eth1 rx off tx off
+    /usr/local/sbin/ethtool -C eth1 adaptive-rx off adaptive-tx off rx-usecs 125
+    /usr/local/sbin/ethtool -G eth1 rx 1024
+    for proto in tcp4 udp4 tcp6 udp6; do
+        echo "/usr/local/sbin/ethtool -N eth1 rx-flow-hash $proto sdfn"
+        /usr/local/sbin/ethtool -N eth1 rx-flow-hash $proto sdfn
+    done
+
+In the example above the `set_irq_affinity` script is used form the NIC drivers sources.
+In the cpu affinity section of suricata.yaml config :
+
+::
+
+ # Suricata is multi-threaded. Here the threading can be influenced.
+ threading:
+  set-cpu-affinity: yes
+  # Tune cpu affinity of threads. Each family of threads can be bound
+  # on specific CPUs.
+  #
+  # These 2 apply to the all runmodes:
+  # management-cpu-set is used for flow timeout handling, counters
+  # worker-cpu-set is used for 'worker' threads
+  #
+  # Additionally, for autofp these apply:
+  # receive-cpu-set is used for capture threads
+  # verdict-cpu-set is used for IPS verdict threads
+  #
+  cpu-affinity:
+    - management-cpu-set:
+        cpu: [ "1-10" ]  # include only these CPUs in affinity settings
+    - receive-cpu-set:
+        cpu: [ "0-10" ]  # include only these CPUs in affinity settings
+    - worker-cpu-set:
+        cpu: [ "18-35", "54-71" ]
+        mode: "exclusive"
+        # Use explicitely 3 threads and don't compute number by using
+        # detect-thread-ratio variable:
+        # threads: 3
+        prio:
+          low: [ 0 ]
+          medium: [ "1" ]
+          high: [ "18-35","54-71" ]
+          default: "high"
+    #- verdict-cpu-set:
+    #    cpu: [ 0 ]
+    #    prio:
+    #      default: "high"
+
+In the af-packet section of suricata.yaml config :
+
+::
+
+  - interface: eth1
+    # Number of receive threads. "auto" uses the number of cores
+    threads: 18 
+    cluster-id: 99
+    cluster-type: cluster_qm
+    defrag: no
+    use-mmap: yes
+    mmap-locked: yes
+    tpacket-v3: yes
+    ring-size: 100000
+    block-size: 1048576
+  - interface: eth1
+    # Number of receive threads. "auto" uses the number of cores
+    threads: 18 
+    cluster-id: 99
+    cluster-type: cluster_qm
+    defrag: no
+    use-mmap: yes
+    mmap-locked: yes
+    tpacket-v3: yes
+    ring-size: 100000
+    block-size: 1048576
+
```

Comment:
maybe explain why two sections are used and maybe a bit about why you would recommend those values